src/pages/new-chat/new-chat.ts

```diff
@@ -41,28 +52,59 @@ export class NewChatComponent implements OnInit {
     });
   }
 
-  private findUsers(): ObservableCursor<User> {
+  findContacts(): Promise<Contact[]> {
+    // If we're running this in the browser, don't look for any contacts
+    if (!navigator.hasOwnProperty('contacts')) return Promise.resolve();
```

Comment:
Can you please explain why `!navigator.hasOwnProperty('contacts')` is the way to test if you are in a browser? 
Ionic docs suggest to use `Platform`: https://ionicframework.com/docs/v2/api/platform/Platform/